src/black_primer/primer.json

```
@@ -75,6 +75,13 @@
       "long_checkout": false,
       "py_versions": ["all"]
     },
+    "pyanalyze": {
+      "cli_arguments": [],
```

I think we need to add `--experimental-string-processing` here
```suggestion
      "cli_arguments": ["--experimental-string-processing"],
```

---

It's in the pyproject.toml, does that not get picked up?